node.js

```diff
@@ -1,18 +1,17 @@
 "use strict";
 
 module.exports = {
-  extends: "./index.js",
+  extends: ["./index.js", "plugin:node/recommended"],
   env: {
     node: true,
-    es6: true,
-  },
-  parserOptions: {
-    ecmaVersion: 2019,
+    es2020: true,
```

Comment:
1. Is there a reason `parserOptions` was removed?  Is it no longer needed?
2. Isn't `env.es2020:true` already inherited from index.js?

Reply:
1. As per the [docs](https://eslint.org/docs/user-guide/configuring/language-options#specifying-environments), setting the ECMA version in the `env` object will automatically assign the corresponding `ecmaVersion` in the `parserOptions`.
2. You're right, this can be removed from both "child" configurations.